ajax/libs/event-drops/package.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "event-drops",
+	"filename": "src/index.min.js",
+	"version": "0.3.1-alpha3",
+	"description": "A time based / event series interactive visualization using d3.js. Use drag and zoom to navigate in time.",
+	"main": "dist/index.js",
+	"homepage": "https://github.com/marmelab/EventDrops",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/marmelab/EventDrops"
+	},
+	"author": "Thiery Michel <thiery@marmelab.com>",
+	"license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/marmelab/EventDrops.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "index.js",
+          "config.js",
+          "fiterData.js",
+          "meatballs.js",
+          "xAxis.js",
+          "zoom.js",
+          "style.css",
+          "drawer/delimiters.js",
+          "drawer/drops.js",
+          "drawer/index.js",
+          "drawer/labels.js",
+          "drawer/lineSeparator.js",
+          "drawer/xAxis.js"
```

Comment:
Do we need all these files? I think we should use npm auto-update here and grab the whole `dist` folder.